### PR TITLE
Adds logging in the triggering monitor situation

### PR DESCRIPTION
**Pull request by Dominik1999**
Adds logging when the MS scheduled for triggering monitor but didn't at the end

### CODE REVIEW

File: src/monitoring_service/handlers.py [hunk 1]
```diff
@@ -381,16 +381,19 @@ def action_monitoring_triggered_event_handler(event: Event, context: Context) ->
         non_closing_signer=event.non_closing_participant,
     )
     if monitor_request is None:
+        log.error("MonitorRequest was lost or deleted before triggering", monitor_request=monitor_request)
```
Comment (palango) [inline src/monitoring_service/handlers.py]: I'd write something like: "MonitorRequest cannot be found"

Also, it doesn't make sense to log `monitor_request` as it is `None`, but I'd log `token_network_address` and `channel_id`.

File: src/monitoring_service/handlers.py [hunk 2]
```diff
@@ -381,16 +381,19 @@ def action_monitoring_triggered_event_handler(event: Event, context: Context) ->
...
         return
 
     channel = context.db.get_channel(
         token_network_address=monitor_request.token_network_address,
         channel_id=monitor_request.channel_identifier,
     )
     if channel is None:
+        log.error("Channel info was lost or deleted before triggering", monitor_request=monitor_request, channel=channel)
```
Comment (palango) [inline src/monitoring_service/handlers.py]: Same as above, just put that the channel info isn't there, and logging channel doesn't make sense

File: src/monitoring_service/handlers.py [hunk 3]
```diff
@@ -381,16 +381,19 @@ def action_monitoring_triggered_event_handler(event: Event, context: Context) ->
...
         return
 
     if not _is_mr_valid(monitor_request, channel):
+        log.error("MonitorRequest lost its validity, this is an error!", monitor_request=monitor_request, channel=channel)
```
Comment (palango) [inline src/monitoring_service/handlers.py]: No need for "this is an error"

File: src/monitoring_service/handlers.py (hunk 1 above)
Comment (Dominik1999) [reply src/monitoring_service/handlers.py]: haha true

**Pull request merged**